tests/Jellyfin.MediaEncoding.Tests/Probing/ProbeResultNormalizerTests.cs

```diff
@@ -25,6 +25,7 @@ public class ProbeResultNormalizerTests
         [InlineData("120/1", 120f)]
         [InlineData("1704753000/71073479", 23.98578237601117f)]
         [InlineData("0/0", null)]
+        [InlineData("0/1", null)]
```

Comment:
Not gonna lie, this C# testing is kinda alien magic to me coming from Java but I guess this should add the test for the circumstance I am looking for.

Read a bit here if anyone else is as confused as me: https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-dotnet-test